cmd/kafka_consumer_group_exporter.go

```diff
@@ -0,0 +1,77 @@
+package main
+
+import (
+	"context"
+	"net/http"
+	"os"
+	"time"
+
+	log "github.com/Sirupsen/logrus"
+
+	"github.com/kawamuray/prometheus-kafka-consumer-group-exporter/kafka"
+	kafkaprom "github.com/kawamuray/prometheus-kafka-consumer-group-exporter/prometheus"
+	"github.com/kawamuray/prometheus-kafka-consumer-group-exporter/sync"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
+	"github.com/urfave/cli"
+)
+
+const consumerGroupCommandName = "kafka-consumer-groups.sh"
+
+func main() {
+	app := cli.NewApp()
+	app.Name = "kafka_consumer_group_exporter"
+	app.Version = version
+	app.Usage = "[OPTIONS] BOOTSTRAP_SERVER"
+	app.Flags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "consumer-group-command-path",
+			Usage: "Path to `kafka-consumer-groups.sh`.",
+			Value: consumerGroupCommandName,
+		},
+		cli.StringFlag{
+			Name:  "listen",
+			Usage: "Interface and port to listen on.",
+			Value: ":7979",
+		},
+		cli.DurationFlag{
+			Name:  "kafka-command-timeout",
+			Usage: "The maximum time the Kafka command is allowed to take before we kill it. We've seen it block forever in production at times (most likely during rebalances).",
+			Value: 5 * time.Minute,
+		},
+		cli.IntFlag{
+			Name:  "max-concurrent-group-queries",
+			Usage: "The maximum number of consumer groups that are queried concurrently.",
+			Value: 20,
```

Comment:
> Do you think that's a sane limit?

https://github.com/apache/kafka/blob/trunk/bin/kafka-run-class.sh#L210
Given that Kafka defaults maximum heap size to 256M for commands, `20` sounds little dangerous depending on where users runs this exporter.
I don't think consumer group command really allocates 256MB on each run but potentially it means this exporter can allocate `256M * 20 = 5GB` at maximum just for reporting metrics, which might lead memory contention with other processes that are more important.
The worst case I imagine here is that some users run this exporter on a server which also runs application processes and this exporter disturbs its memory allocation periodically ;;
We need to take a balance between safety and better usability, so I'd suggest to set this default to `4`, which potentially allocates memory up to 1G+ bytes by default.
WDYT?